chore: react/vue tutorials and translations reworded

jonniebigodes opened the pull request:
With this pr the issue #153 is addressed in it's both variants. The initial cause for the issue and what was brough to the attention with the [following comment](https://github.com/chromaui/learnstorybook.com/issues/153#issuecomment-584207942)

What was done:

Going by Framework
- React
    - Conclusion
  Both english and portuguese versions include a mention to the discord channel and blog for better discoverability.
    - Data 
    In both english and portuguese versions the reference to package json when installing redux was removed. Expanded the ending note with a mention to run the tests with the flag that is introduced in the end of the get-started section.
   - Deploy
      shortened and with that simplified the exporting as a static app section, GitHub section was reworded to match the existing versions. As they abide by the same convention. Removed lingering presences of the `.storybook-static` folder, the portuguese translation follows suit.
   - Get Started
     Added some wording to address the issue that kickstarted this whole endeavour. The portuguese translation follows suit.
   - Screen
     Updated `app.js` code snippet to match the current one. The portuguese translation follows suit.
   - Simple Component
      Unified the whole code snippets to a single approach. Updated the `.storybook/main.js` code snippet to it's most accurate one. The portuguese translation was updated to match the current one.
   - Test
      Moved chromatic instalation reference to a development dependency per convention. The portuguese translation follows suit.
    - Using Addons
       Moved knobs to a development dependency and updated `.storybook/main.js` to the most accurate one. Once again the portuguese translation was adjusted to match version 5.3 and these changes.
   - Some of the translations, more specifically: es, zh-cn, zh-tw were changed to include a note mentioning that the translation is not accurate. The wording is the equivalent to the one used in the angular version i mentioned in the issue that is currently addressing this. A side note, i used google translator to address both chinese translations, probably the wording might be a bit off and if is, i appologize for that. But i decided to have something there to avoid misleading people as the translation is no longer accurate.
   - Create addons
      - The entire section was reworder to match the current state of Storybook. The code snippets aswell. The pt translation follows suit.

- Vue
  - Get Started
     I did a bit of research and the same approach can be taken with this framework as for React. I added a note mentioning that the reader can alternatively use npm if he's willing to do so. The portuguese translation also covers this.
  - Simple Component
     Fixed some missing factory functions in some code snippets that were brought to the attention in another pr. Removed the Vue reference when dealing with Stateless components. This in order to make the sentence more simple. The reader already knows he's in a Vue related tutorial, no need to insist on it and keep him engaged. The portuguese translation is also updated.
  - Using Addons
     The code snippets were cut down to match the React Version. This was something that i overlooked. And there's no need on having a big code snippet when the intention is to keep the reader's focus on the important item. The portuguese translation is updated aswell.
   - Conclusion
      The same items were introduced here as in the React version in both the english version and portuguese translations.
    - Composite Component
        The portuguese translation code snippets were updated to match the original version.
     - Data
         Code snippets were updated to match the current english version.
     - Deploy
         There is some minor rewording that was made in the portuguese translation


I'd like to bring to the attention of @IvanSotelo as you spearheaded the spanish translation this pr. More specifically the Vue edition. If this goes through if you don't mind adjust accordingly. From the top of my head, the only items of concern are the Get started section, conclusion and using addons. 

Also @renet @jonathanherdt for their fantastic job on translating the React Version of the tutorial. If both of you are up for it, when this pr goes through it would awesome if you both coordinated and make the adjustments to the German translation. 
Also @Robin-Hoodie, @ghengeveld, more specifically @Robin-Hoodie as you spearheaded the Dutch translation, once and if this pr goes through if you're ok with it it would be awesome if you could make the adjustments to the Dutch translation. @ndelangen i'm aware that you're extremely busy working on Storybook but when and if the translation goes through it would be awesome if you once again proof read it. Sounds good to all of you?

Feel free to provide feedback

domyen commented on content/intro-to-storybook/react/en/conclusion.md:
```diff
@@ -21,6 +21,10 @@ Want to dive deeper? Here are helpful resources.
 
 - [**Visual Testing Handbook**](https://www.chromaticqa.com/book/visual-testing-handbook) dives deep into using Storybook to visual test components. Free 31-page ebook.
 
+- [**Storybook Discord**](https://discord.gg/UUt2PJb) will put you in touch with the Storybook community. Not only offers a means to help you but for you to help the community also.
```
```suggestion
- [**Storybook Discord chat**](https://discord.gg/UUt2PJb) puts you in contact with the Storybook community. Get and give help to other Storybook users.
```

jonniebigodes replied:
@domyen marking this as solved per latest commit i just did

domyen commented on content/intro-to-storybook/react/en/conclusion.md:
```diff
@@ -21,6 +21,10 @@ Want to dive deeper? Here are helpful resources.
 
+- [**Storybook Discord**](https://discord.gg/UUt2PJb) will put you in touch with the Storybook community. Not only offers a means to help you but for you to help the community also.
+
+- [**Storybook blog**](https://medium.com/storybookjs) where you'll read about the latest changes and some techniques to further improve your development workflows with Storybook.
```
```suggestion
- [**Storybook blog**](https://medium.com/storybookjs) showcases the latest releases and features to streamline your UI development workflow.
```

jonniebigodes replied:
@domyen thank you for the suggestions. I struggled a bit with these 2 items, to better convey the message. But the change looks real nice simple , straight to the point

jonniebigodes replied:
@domyen marking this as solved per latest commit

tmeasday commented:
Sorry I haven't got to this yet @jonniebigodes, it is definitely on my list

jonniebigodes commented:
@tmeasday no worries...no worries, I still need to addres the changes proposed by @domyen

jonniebigodes commented:
With the latest commit the create addons section got the much deserved attention that was required. I introduced a couple of changes in the wording and adjusted the code snippets accordingly to match the current version of Storybook. And introduced a the final piece in the pt translation to make it whole.

The rest of the create addons sections will require some extra special attention as per the issue i've created. I'm currently going over it and i'll make a pr to address the other frameworks. 

Feel free to provide feedback

tmeasday commented on content/intro-to-storybook/react/en/conclusion.md:
```diff
@@ -9,7 +9,7 @@ Congratulations! You created your first UI in Storybook. Along the way you learn
 [🌎 **Deployed Storybook**](https://clever-banach-415c03.netlify.com/)
 
-Storybook is a powerful tool for React, Vue, and Angular. It has a thriving developer community and a wealth of addons. This introduction scratches the surface of what’s possible. We’re confident that once you adopt Storybook, you’ll be impressed by how productive it is to build durable UIs.
+Storybook is a powerful tool React, React Native, Vue, Angular, Svelte and many others frameworks. It has a thriving developer community and a wealth of addons. This introduction scratches the surface of what’s possible. We’re confident that once you adopt Storybook, you’ll be impressed by how productive it is to build durable UIs.
```
```suggestion
Storybook is a powerful tool for React, React Native, Vue, Angular, Svelte and many others frameworks. It has a thriving developer community and a wealth of addons. This introduction scratches the surface of what’s possible. We’re confident that once you adopt Storybook, you’ll be impressed by how productive it is to build durable UIs.
```

jonniebigodes replied:
@tmeasday i forgot to commit this. i had the change in play but forgot to add the file. Going to mark it as resolved per latest commit.

tmeasday commented on content/intro-to-storybook/react/en/creating-addons.md:
```diff
@@ -38,81 +35,62 @@ export default {
-addons.register("my/design-assets", () => {
-  addons.add("design-assets/panel", {
-    title: "assets",
+//.storybook/addons/register.js
```
Should we call this file `design-addon/register.js` maybe? (I assume it needs to be called `.../register.js`)

jonniebigodes replied:
@tmeasday sounds a bit more clearer to what's intended with the section. I've updated the snippets to reflect this with the latest commit. I'm going to mark it as resolved.

jonniebigodes commented:
@tmeasday if you don't mind taking a look at the changes. If it's ok with you, i'll merge it and close the underlying issue associated with this. Sounds good?

tmeasday commented:
LGTM thanks @jonniebigodes
